Casks/r/removebg.rb

```diff
@@ -7,9 +7,11 @@
   desc "Automatic bulk background removal"
   homepage "https://www.remove.bg/"
 
+  # The in-app updater returns a JSON file, but it is not a valid JSON file
+  # so we check the response with a regex using the :page_match strategy instead
```

Comment:
```suggestion
  # The in-app updater checks a JSON file but it has syntax issues, so we can't
  # use the `Json` strategy until this is resolved.
```

Just some wording tweaks for the sake of flow/clarity. For what it's worth, I use the class name when talking about strategies (e.g., `PageMatch`) instead of the `#strategy` symbol (e.g., `:page_match`), as the latter is just a reference.